lib/Amazon/MiqEc2InstanceTypes.rb

```diff
@@ -11,6 +11,8 @@
 #     NOTE: We may want to consider using the source data directly, however
 #           there aren't the discontinued types.
 #           https://raw.githubusercontent.com/powdahound/ec2instances.info/master/www/instances.json
+#           http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html
+#           http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/c4-instances.html
```

Comment:
Minor, but can you move these URLs up to the other URLs.  This particular URL that you appended to is part of that NOTE, and thus don't make sense here.